### DISCUSSION TITLE
Wallet connection issue with multiple full nodes on the same network

### DISCUSSION THREAD

**Discussion by milan-patartics** [Chia-Network/chia-blockchain#3510] [Support]
I have two full nodes, **A** and **B**. Both are at the same network with the same public IP. At the moment I am plotting and farming on both at the same time until I'll have my dedicated farmer node set up and running. They are working, but I noticed a strange thing I would like to understand.

I started node **A** a few minutes before **B**. **A** has **one** connection the the wallet (localhost) with a few MiB of upstream traffic. But node **B** has 0 connection to the wallet (also localhost) and more than 9000 MiB of upstream traffic. Why does this happen?

I am just guessing, but seems like the network allows only one wallet connection per key at the same time. From a security perspective it could be useful, but is it really the case? If it is true, doesn't that prevent using a wallet from other locations when you have a full node for a specific key running?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
